src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -264,16 +264,20 @@ void P104_data_struct::configureZones() {
   String log;
 
   if (loglevelActiveFor(LOG_LEVEL_INFO) &&
-      log.reserve(32)) {
+      log.reserve(45)) {
     log  = F("P104: configureZones to do: ");
     log += zones.size();
     addLog(LOG_LEVEL_INFO, log);
   }
   # endif // ifdef P104_DEBUG
 
+  P->displayClear();
+  
   for (auto idx = zones.begin(); idx != zones.end(); ++idx) {
     if ((*idx).zone <= expectedZones) {
```

Comment:
why `(*idx).zone` instead of `idx->zone` ?

Reply:
Mostly lack of experience as a c++ developer, I guess. I'll fix & rename 👍

Reply:
> Mostly lack of experience as a c++ developer, I guess. I'll fix & rename 👍

Well you'll now getting the 10 years drill I got at a previous job in a very short time... and you can keep up with it too.
So you're learning fast :)

Reply:
Thanks, the feedback is really helpful, and it may help that I'm getting feedback for a looong time already 🤣 (aka: 'Software is never finished, there is always room for improvement')